src/onepasswordconnectsdk/async_client.py

```diff
@@ -71,7 +71,7 @@ async def get_file_content(self, file_id: str, item_id: str, vault_id: str, cont
         return response.content
 
     async def download_file(self, file_id: str, item_id: str, vault_id: str, path: str):
-        file_object = await self.get_file(file_id, item_id, vault_id)
+        file_object = await self.get_file(file_id, item_id, vault_id) 
```

Comment:
Looks like a space was accidentally added here, let's remove this.

Reply:
Looks like I was in the area, and then forgot to add the return type 😅